DigitalLearningSolutions.Data.Tests/DigitalLearningSolutions.Data.Tests.csproj

```diff
@@ -7,6 +7,10 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <ItemGroup>

```

Comment:
I'm not sure why this has  been added. It seems to reference a path on your local machine.